aca_hi_bgd/update_bgd_events.py

```diff
@@ -1236,12 +1236,17 @@ def main(args=None):  # noqa: PLR0912, PLR0915 too many branches, too many state
                 outdir=event_outdir,
             )
             LOGGER.warning(f"HI BGD event in obsid {obsid} {url}")
-            if len(opt.emails) > 0:
+
+            # Add another filter on the data for the emails to only include
+            # events with more than 200 slot seconds.
+            if np.any(obs_events["slot_seconds"] > 200) and len(opt.emails) > 0:
```

Comment:
This threshold would probably be better as a configurable item someplace, but it seems just as easy to find here where the emails are actually sent.